pkg/utils/managedresources/managedresources.go

```diff
@@ -220,7 +221,14 @@ func WaitUntilListDeleted(ctx context.Context, client client.Client, mrList *res
 			if resourcesAppliedCondition != nil && resourcesAppliedCondition.Status != gardencorev1beta1.ConditionTrue &&
 				(resourcesAppliedCondition.Reason == resourcesv1alpha1.ConditionDeletionFailed || resourcesAppliedCondition.Reason == resourcesv1alpha1.ConditionDeletionPending) {
 				deleteError := fmt.Errorf("%w:\n%s", err, resourcesAppliedCondition.Message)
```

Comment:
Nit: you could move this inside the below `var()`

Reply:
I don't see the need of it.